### ISSUE TITLE
Autosave every x seconds

### ISSUE THREAD

**Issue by founderio** [stringandstickytape/RegulatedNoise#55]
A good enhancement for compatibility with other tools would be to add an option, that makes RegulatedNoise write out its AutoSave.csv (or any other file for that matter) every x seconds so other tools sourcing that data can auto-update..

Currently, as far as I can tell that only happens on close or manually (with a file dialog every time). 
Could save users much time and should be quite easy I assume.

Also an alternative would be to have another option for auto-saving after OCR.

**Comment by Duke-Jones** [comment]
It is done in the international version.
I hope it will be included in this main version.

**Comment by founderio** [comment]
So we are waiting on #57, right?

**Comment by Duke-Jones** [comment]
yes, I hope so but stringandstickytape seems to be very busily with other things.
Until this is done your are invited try out my parallel version:

https://github.com/Duke-Jones/RegulatedNoise/releases
